core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3846,7 +3846,7 @@ gOptWDiags("W", llvm::cl::Prefix, llvm::cl::ZeroOrMore,
 // Really OneOrMore, will be changed in RootClingMain below.
 static llvm::cl::list<std::string>
 gOptDictionaryHeaderFiles(llvm::cl::Positional, llvm::cl::ZeroOrMore,
-                         llvm::cl::desc("<list of dictionary header files> <LinkDef file>"),
+                         llvm::cl::desc("<list of dictionary header files> <LinkDef file | selection xml file>"),
```

Comment:
One thing to remember is that rootcling and genreflex operate in slightly different modes. If we ask rootcling to handle a selection.xml files, we will need to also pass an option `--reflex` to enable the reflex extensions...

Reply:
This is an interesting point. Do we want rootcling to behave like rootcling or like genreflex in presence of a selection xml?

Reply:
So currently the `reflex` option is 
```
gOptReflex("reflex", llvm::cl::desc("Deprecated, legacy flag which is ignored."),
          llvm::cl::Hidden,
          llvm::cl::cat(gRootclingOptions));
```
We can imagine a new option name perhaps?

Reply:
Could be. We can remove the deprecation as that’s a real use case.